.gitlab-ci.yml

```diff
@@ -223,500 +215,221 @@ before_script:
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_contiv_sep_variables: &ubuntu_contiv_sep_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.coreos_cilium_variables: &coreos_cilium_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_cilium_sep_variables: &ubuntu_cilium_sep_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.rhel7_weave_variables: &rhel7_weave_variables
-# stage: deploy-part1
-  MOVED_TO_GROUP_VARS: "true"
-
-.centos7_flannel_addons_variables: &centos7_flannel_addons_variables
-# stage: deploy-part2
-  MOVED_TO_GROUP_VARS: "true"
-
-.debian9_calico_variables: &debian9_calico_variables
-# stage: deploy-part2
-  MOVED_TO_GROUP_VARS: "true"
-
-.coreos_canal_variables: &coreos_canal_variables
-# stage: deploy-part2
-  MOVED_TO_GROUP_VARS: "true"
-
-.rhel7_canal_sep_variables: &rhel7_canal_sep_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_weave_sep_variables: &ubuntu_weave_sep_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.centos7_calico_ha_variables: &centos7_calico_ha_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.centos7_kube_router_variables: &centos7_kube_router_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.centos7_multus_calico_variables: &centos7_multus_calico_variables
-# stage: deploy-part2
-  UPGRADE_TEST: "graceful"
-
-.coreos_alpha_weave_ha_variables: &coreos_alpha_weave_ha_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.coreos_kube_router_variables: &coreos_kube_router_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_rkt_sep_variables: &ubuntu_rkt_sep_variables
-# stage: deploy-part1
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_flannel_variables: &ubuntu_flannel_variables
-# stage: deploy-part2
-  MOVED_TO_GROUP_VARS: "true"
-
-.ubuntu_kube_router_variables: &ubuntu_kube_router_variables
-# stage: deploy-special
-  MOVED_TO_GROUP_VARS: "true"
-
-.opensuse_canal_variables: &opensuse_canal_variables
-# stage: deploy-part2
-  MOVED_TO_GROUP_VARS: "true"
 
+.packet: &packet
+  <<: *packet-job
+  <<: *testcases
+  variables:
+    <<: *packet_variables
 
 # Builds for PRs only (premoderated by unit-tests step) and triggers (auto)
+
 ### PR JOBS PART1
 
-gce_ubuntu18-flannel-aio:
+packet_ubuntu18-flannel-aio:
   stage: deploy-part1
-  <<: *job
-  <<: *gce
-  variables:
-    <<: *ubuntu18_flannel_aio_variables
-    <<: *gce_variables
+  <<: *packet
   when: on_success
   except: ['triggers']
-  only: [/^pr-.*$/]
+  only: ['master', /^pr-.*$/]
 
-### PR JOBS PART2
+# ### PR JOBS PART2
 
 gce_coreos-calico-aio:
   stage: deploy-part2
-  <<: *job
   <<: *gce
-  variables:
-    <<: *coreos_calico_aio_variables
-    <<: *gce_variables
   when: on_success
   except: ['triggers']
   only: [/^pr-.*$/]
 
-gce_centos7-flannel-addons:
+packet_centos7-flannel-addons:
```

Comment:
They are not exclusive. Let's keep both, add packet jobs without removing any gce.

Reply:
I was asked to move ci jobs to packet, not copy. feel free to add gce again if you want.